_posts/2025-04-21-OpenSearch-Becnhmark--An-Update.md

```diff
@@ -0,0 +1,58 @@
+---
+layout: post
+title: OpenSearch Benchmark: An update
+authors:
+- govkamat
+date: 2025-04-21 12:30:00 -0600
+categories:
+  - community
+excerpt: OpenSearch Benchmark is the performance benchmarking tool for OpenSearch.
+meta_keywords: OpenSearch Benchmark, workloads, k-NN, big5
+meta_description: OpenSearch Benchmark has come a long way, and there is so much more ahead.
```

Comment:
Please update the meta with the following: 

_meta_keywords:_ OpenSearch benchmark, performance testing, benchmarking tool, big5 workloads, OpenSearch benchmark roadmap

_meta_description:_ Explore OpenSearch Benchmark's latest updates, including new generative AI workloads and enhanced testing features. Learn how this performance testing tool helps optimize OpenSearch deployments and ways to contribute.